client/components/redirect.jsx

```diff
@@ -0,0 +1,10 @@
+import { useHistory } from 'react-router-dom';
+
+export default function Redirect(props) {
+  const history = useHistory();
+  if (props.to === '') {
+    history.push('/');
+  } else {
+    history.push(props.to);
+  }
+}
```

Comment:
There's a component for this in react router.
https://reactrouter.com/web/api/Redirect